tee-worker/ts-tests/common/utils/identity.ts

```diff
@@ -0,0 +1,224 @@
+import { ApiPromise } from '@polkadot/api';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { Codec } from '@polkadot/types/types';
+import { HexString } from '@polkadot/util/types';
+import { hexToU8a, u8aToHex } from '@polkadot/util';
+import { blake2AsHex } from '@polkadot/util-crypto';
+import {
+    EvmNetwork,
+    IdentityGenericEvent,
+    IntegrationTestContext,
+    LitentryIdentity,
+    LitentryValidationData,
+    SubstrateNetwork,
+    Web2Network,
+} from '../type-definitions';
+import { decryptWithAES, encryptWithTeeShieldingKey } from './crypto';
+import { ApiTypes, SubmittableExtrinsic } from '@polkadot/api/types';
+
+const base58 = require('micro-base58');
+
+export async function createTrustedCallSigned(
```

Comment:
this is a dead code, can we get rid of it ?
also i doubt it's specific to identity.

Reply:
Yeap,we can get rid of it, the use case update is [here](https://github.com/litentry/litentry-parachain/blob/1668-split-test-utils/tee-worker/ts-tests/examples/direct-invocation/util.ts#L58)